core/src/scene/scene.h

```diff
@@ -32,6 +33,10 @@ class Scene {
     Scene();
     ~Scene();
 
+    using ArrayValue = std::vector<double>;
+    using Value = variant<none_type, bool, double, std::string, ArrayValue>;
+    std::unordered_map<std::string, Value> m_globals;
```

Comment:
In the current implementation, we actually do not need scene to store a reference of all the parsed globals. But we will need it once we add dynamic updates to scene!